mappings/net/minecraft/world/dimension/DimensionType.mapping

```diff
@@ -98,5 +98,6 @@ CLASS net/minecraft/class_2874 net/minecraft/world/dimension/DimensionType
 	METHOD method_29961 getInfiniburnBlocks ()Lnet/minecraft/class_3494;
 	METHOD method_31108 equals (Lnet/minecraft/class_2874;)Z
 		ARG 1 dimensionType
+	METHOD method_31109 getCoordinateScaleRatio (Lnet/minecraft/class_2874;Lnet/minecraft/class_2874;)D
```

Comment:
Isn't a scale already a ratio?

Reply:
This is getting the ratio between the coordinate scales of two dimensions. Maybe `getNetCoordinateScale` is a better name?

Reply:
Is this directly multiplied with a coordinate? Then potentially getCoordinateScaleFactor?

Reply:
No, it's not multiplied with a coordinate itself.

Reply:
I went with `getCoordinateScaleFactor`.